### PR TITLE
[US-684] Example of preparing chart report from CSV

### PR DESCRIPTION
**Pull request by kellemNegasi**

### CODE REVIEW

**Comment by anovik** [comment]
@kellemNegasi is this ready for review? If so, can you add reviewers, now it should work correctly in this repo

File: spreadsheet/multiple-charts-from-csv/main.go [hunk 1]
```diff
@@ -0,0 +1,212 @@
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
```
Comment (anovik) [inline spreadsheet/multiple-charts-from-csv/main.go]: Should the copyright year be 2024?

File: spreadsheet/multiple-charts-from-csv/main.go [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+	QuantitySold    float64
+	SalePrice       float64
+	SalespersonName string
+}
+
+func main() {
+	filePath := "./data/test-data.csv"
+	data, header, err := load_csv(filePath)
+	if err != nil {
+		panic("failed to load data ")
```
Comment (anovik) [inline spreadsheet/multiple-charts-from-csv/main.go]: could you please change panic to error logging?

File: spreadsheet/multiple-charts-from-csv/main.go [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+		cumulativeSumsByPerson[dataRow.SalespersonName] += dataRow.SalePrice
+		row.AddCell().SetNumber(cumulativeSumsByPerson[dataRow.SalespersonName])
+	}
+
+	drawing := ss.AddDrawing()
+
+	// Create Line Chart showing daily sales.
+	chart, anc := drawing.AddChart(spreadsheet.AnchorTypeTwoCell)
+	// Set width to 8 cells
+	anc.SetWidthCells(10)
```
Comment (anovik) [inline spreadsheet/multiple-charts-from-csv/main.go:78]: in comment 8 cells, in code 10, why?

**Review by anovik — COMMENTED** [review]
Left some comments

File: spreadsheet/multiple-charts-from-csv/main.go (hunk 3 above)
Comment (kellemNegasi) [reply spreadsheet/multiple-charts-from-csv/main.go:78]: I have also updated this comment, I am not sure why it is not updating here.

**Review by 3ace — APPROVED** [review]

**Review by sampila — APPROVED** [review]
Looks good to me

**Pull request merged**